contracts/ERC1155NFT.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract ERC1155NFT is ERC1155, Ownable {
```

Comment:
Instead of Ownable we have started using AccessControl, so we can deploy the contract with only minimal roles to ourselves (just minting pretty much) and giving the user thats deploying the contract more access rights, example in "contracts/NftInjectableTokenIdRoles.sol". But since using AccessControl Opensea for example did not give the deploying user editing rights for the contract on OpenSea, to solve this we've made a "function owner()" (Same signature as Ownable owner function) so it just shows up in Opensea for the users who deployed it

Reply:
Solved